mlflow/projects/utils.py

```diff
@@ -245,3 +253,74 @@ def _create_run(uri, experiment_id, work_dir, version, entry_point, parameters):
                    list(final_params.items()) + list(extra_params.items())]
     tracking.MlflowClient().log_batch(active_run.info.run_id, params=params_list)
     return active_run
+
+
+def get_conda_command(conda_env_name):
```

Comment:
Given that this method is also called by `mlflow models serve` and other APIs unrelated to projects, IMO it'd be better to put them in a `mlflow/utils/conda_utils.py` module - what do you think? The thinking is that it's a bit odd to have modules under `mlflow.pyfunc` depend on `mlflow.projects.utils` - thanks! Same goes for `get_conda_bin_executable`

Reply:
I don't not see the usage of this method in scoring server, but I agree about to put this method in a specific module.